### ISSUE TITLE
SlevomatCodingStandard.Commenting.AnnotationName reports Doctrine annotations

### ISSUE THREAD

**Issue by mabar** [slevomat/coding-standard#1558]
For following code I get reports `Annotation name is incorrect. Expected @before, found @Before.` and `Annotation name is incorrect. Expected @after, found @After.`

It does not break code - tests and static analysis still pass, but phpstorm reports classes as unused and it looks bit ugly when used together with other PascalCase annotations

```php
use Orisai\ObjectMapper\Attributes\Callbacks\After;
use Orisai\ObjectMapper\Attributes\Callbacks\Before;

/**
 * @Before("before")
 * @After("after")
 */
trait A
{

	private function before(): void
	{
		// Noop
	}

	private function after(): void
	{
		// Noop
	}

}
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/cf10256483cd85e4fe77ed5896b66a287cd8f481
